ptx/sec_limit_continuity.ptx

```diff
@@ -217,12 +219,14 @@
             \end{tikzpicture}
           </latex-image>
           <description>
-            Shows the domain -2.5 to 3.5 and range -2.5 to 2.5. There are five
-            straight lines, each parallel with the x-axis. Each of the lines is
-            one unit in length and undefined on its right side, but defined on 
-            their left. Line one is defined by the points <m>(-2, -2), (-1, -2)</m>,
-            line two <m>(-1, -1), (0, -1)</m>, line 3 <m>(0, 0), (1, 0)</m>, line 4 
-            <m>(1, 1), (2, 1)</m>, and line 5 <m>(2, 2), (3, 2)</m>.
+            <p>
+              Shows the domain -2.5 to 3.5 and range -2.5 to 2.5. There are five
```

Comment:
should have math tags around each number, and around x (`<m>x</m> axis` rather than `x-axis`)

Reply:
In previous sections of the chapter I have used x-axis and y-axis, instead of just making the changes here I'd like to get them all in one go. Would that work with you?

Reply:
Sorry for the slow reply -- I've been busy getting grades in for this term.

I'll merge with x-axis, etc. and if you want to make another PR later that changes them all, that would be great.

Reply:
That's fine, don't worry about it. I'll make another PR with the changes